examples/bimodal_ke/main.rs

```diff
@@ -89,6 +89,9 @@ impl<'a> Predict<'a> for Ode {
         time: f64,
         next_time: f64,
     ) -> State {
+        if time == next_time {
+            return x;
+        }
```

Comment:
@Siel, does this look like an OK way to handle events with the same time?

Reply:
It looks fine. I'm still wondering if those abstractions I wrote are the right ones, tho.